src/Plugin/Journal.php

```diff
@@ -25,7 +24,7 @@ public function addSuccess(RequestInterface $request, ResponseInterface $respons
      * Record a failed call.
      *
      * @param RequestInterface $request   Request use to make the call
-     * @param Exception        $exception Exception returned by the call
+     * @param \Exception       $exception Exception returned by the call
      */
-    public function addFailure(RequestInterface $request, Exception $exception);
+    public function addFailure(RequestInterface $request, \Exception $exception);
```

Comment:
Here is the BC break. Possible solution to avoid this is to introduce a new Journal interface. Could we do something better?